zig-http-server/src/main.zig

```diff
@@ -57,8 +66,7 @@ fn handleRequest(response: *http.Server.Response, allocator: std.mem.Allocator)
         // Write the response body.
         try response.do();
         if (response.request.method != .HEAD) {
-            try response.writeAll("Zig ");
-            try response.writeAll("Bits!\n");
+            try response.writeAll("Zig Bits!\n");
```

Comment:
since nagle's algorithm is disabled now each write (since std.http doesn't buffer writes) is an individual tcp packet which is almost guaranteed to force each client to make 2 read calls since this is over localhost. Merging the writes retains the original pre-nodelay semantics (since that's why nagle's algorithm does).